sbt-tzdb/src/main/scala/io/github/sbt/tzdb/IOTasks.scala

```diff
@@ -49,30 +49,29 @@ object IOTasks {
     }
 
   def generateTZDataSources(
-    base:        File,
-    data:        File,
-    log:         Logger,
-    includeTTBP: Boolean,
-    jsOptimized: Boolean,
-    zonesFilter: String => Boolean
+    base:         File,
+    data:         File,
+    log:          Logger,
+    includeTTBP:  Boolean,
+    tzdbPlatform: TzdbPlugin.Platform,
```

Comment:
Cool. I guess this as an API breaking change but that's ok. we just need to increase the major version number

Reply:
:+1: